include/zephyr/bluetooth/hci_types.h

```diff
@@ -3348,6 +3348,13 @@ struct bt_hci_evt_le_cis_req {
 	uint8_t  cis_id;
 } __packed;
 
+#define BT_HCI_LE_BIG_HANDLE_MIN            0x00
+#define BT_HCI_LE_BIG_HANDLE_MAX            0xEF
+#define BT_HCI_LE_BIG_SYNC_DELAY_MIN        0x000030U
+#define BT_HCI_LE_BIG_SYNC_DELAY_MAX        0x7FFFFFU
+#define BT_HCI_LE_TRANSPORT_LATENCY_BIG_MIN 0x000030U
+#define BT_HCI_LE_TRANSPORT_LATENCY_BIG_MAX 0x7FFFFFU
```

Comment:
Just in case someone questions the inconsistent prefix; this is what they are called in the BT spec:
![image](https://github.com/user-attachments/assets/54958503-cc8c-474b-8c76-e9dcdc492a61)